scripts/zones/Windurst_Woods/npcs/Ponono.lua

```diff
@@ -11,34 +14,57 @@ require("scripts/globals/status")
 
 function onTrade(player,npc,trade)
     local newRank = tradeTestItem(player,npc,trade,tpz.skill.CLOTHCRAFT)
-
+    local moralmanifest = player:getQuestStatus(OTHER_AREAS_LOG,tpz.quest.id.otherAreas.A_MORAL_MANIFEST)
+    local tradeGil = trade:getGil()
     if newRank ~= 0 then
         player:setSkillRank(tpz.skill.CLOTHCRAFT,newRank)
         player:startEvent(10012,0,0,0,0,newRank)
     end
+    if moralmanifest == QUEST_ACCEPTED and player:getCharVar("moral") == 2 then
```

Comment:
conditions can't be trigger in one trade, so use `elseif`

Reply:
Not sure what you mean by this, maybe help me out a bit - this specific is under trigger vice trade. Through test I have found that this works based on if they are in the guild or not. Maybe someone else could help test and see if I am not seeing something correctly?

Reply:
```lua
if stuff then
    code
elseif morestuff then
    more code
end
```
vs
```lua
if stuff then
   code
end
if stuff then
   code
end
```

Reply:
this still needs fixed

Reply:
pardon, I commented at wrong place

Reply:
the way its structured now, your code here potentially tries to start an event after an event may have already started.